package.json

```diff
@@ -23,7 +23,9 @@
     "script/bundle-website.sh",
     "script/bundle-page.sh",
     "script/util/clean.sh",
-    "script/util/lisp-case.sh"
+    "script/util/lisp-case.sh",
```

Comment:
It looks like we need `lisp-case.sh` because it's used by `test-page.sh`. I think this is actually a hangover from an old approach where website names were written in `CamelCase` and had to be converted to `lisp-case` by a script. That convention has gone, so I think we can remove the use of `lisp-case.sh` from `test-page.sh` and just use the value of `WEBSITE` unmodified. Then we can remove `lisp-case.sh` from here and indeed from `script/util` in the Fluid repo, because it will no longer be used anywhere.